grabcut/grabcut.ipynb

```diff
@@ -0,0 +1,120 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Grabcut\n",
+    "\n",
+    "Written by Philipp Rudiger<br>\n",
+    "Created: November 14, 2021<br>\n",
```

Comment:
```suggestion
    "Created: November 16, 2018<br>\n",
```

Alright I'm going to set it to the date when this PR (https://github.com/pyviz-topics/EarthSim/pull/253) was merged in earthsim, some time in November 2018.